docs/rorm/transactions.md

```diff
@@ -94,3 +94,50 @@ pub async fn create_user(
     Ok(())
 }
 ```
+
+## Using a Guard
+
+When a single transaction may be used across function calls,
+or when either a transaction or the database itself may be passed to
+these functions, a `TransactionGuard` can be used. It can be
```

Comment:
Not entirely correct. A transaction guard was designed for allowing to either pass a Database or Transaction. You can still use a `Transaction` across function calls.

Reply:
I've addressed this by removing the part about multiple function calls.